### ISSUE TITLE
direct jQuery handle to existing html

### ISSUE THREAD

**Issue by baum1234** [limonte/sweetalert2#263]
When I'm referencing with e.g. $(".my-custom-modal-content") to an existing html object in the html part I can open the modal window only once. On a second try I get this: 
`Uncaught TypeError: Failed to execute 'appendChild' on 'Node': parameter 1 is not of type 'Node'.`

**Comment by limonte** [comment]
Please include a jsfiddle in order for us to be able to reproduce the issue.

JSFiddle template: https://jsfiddle.net/limon/ad3quksn/

**Comment by limonte** [comment]
I'm closing this issue due to inactivity, add jsfiddle and reopen if needed.

**Comment by baum1234** [comment]
Sorry I forgot about it 💃 

Here is the example: https://jsfiddle.net/ad3quksn/39/
Just klick the button, then try it a second time

**Comment by limonte** [comment]
Thank you @baum1234 for the jsfiddle. The issue is fixed in [v4.2.6](https://github.com/limonte/sweetalert2/releases/tag/v4.2.6)

**Comment by baum1234** [comment]
Thank you :)
Maybe allowing the same jQuery reference in titles? That way it would be possible to use e.g. a checkbox in the title that is created with javascript before.

**Comment by limonte** [comment]
> Maybe allowing the same jQuery reference in titles? That way it would be possible to use e.g. a checkbox in the title that is created with javascript before.

I'm not sure right now this right now, it would be great if you create the new issue for this with maybe more argumentation when it could be needed.

**Comment by baum1234** [comment]
I just found out that your fix removed all handles.

https://jsfiddle.net/ad3quksn/40/
Click on the button and then into the input

**Comment by limonte** [comment]
> I just found out that your fix removed all handles.

I'm not sure I understand you. Could you please specify expected and actual behavior?

**Comment by baum1234** [comment]
If you use the older version, before the fix, it was possible to click into the input and a datepicker (from jquery ui) showed up. Now nothing happens.

I changed the fiddle a little bit: https://jsfiddle.net/ad3quksn/41/
The input outside the hidden container is what I expect and in the hidden container is the actual behavior.

**Comment by limonte** [comment]
Yes, because of copying DOM element instead of moving it to the modal, you need to add the handler to add dynamically the datepicker to that input:

https://jsfiddle.net/ad3quksn/42/

The only thing I added is:

```js
$('body').on('focus', ".swal2-modal .picker", function() {
  $(this).removeClass('hasDatepicker').datepicker();
});  
```

**Comment by baum1234** [comment]
But why did it work in the version before? Just use the version before and my code, everything worked fine.
